lib/IDE/Refactoring.cpp

```diff
@@ -6218,14 +6279,13 @@ class AsyncConverter : private SourceEntityWalker {
   }
 
   void convertNode(ASTNode Node, SourceLoc StartOverride = {},
-                   bool ConvertCalls = true) {
+                   bool ConvertCalls = true,
+                   bool IncludePreceedingComment = true) {
```

Comment:
Nit: preceeding -> preceding

Reply:
Not sure how a double e snuck in there, thanks 🙇